src/bundle/ui-dev/src/modules/universal-discovery/browse.tab.module.js

```diff
@@ -38,4 +38,18 @@ ibexa.addConfig(
     true,
 );
 
+ibexa.addConfig(
+    'adminUiConfig.universalDiscoveryWidget.tabs',
+    [
+        {
+            id: 'image_picker',
```

Comment:
adding the same tab with a different id? could you explain why?
additionally: admin-ui should not have any knowledge about image-picker

Reply:
I removed it. The backend part for overwriting config will be done in a separate PR.